src/splitter-stop/SplitterStopSpell.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-FileCopyrightText: © 2024 Dai Foundation <www.daifoundation.org>
+// SPDX-License-Identifier: AGPL-3.0-or-later
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+pragma solidity ^0.8.16;
+
+import {DssEmergencySpell} from "../DssEmergencySpell.sol";
+
+interface SplitterMomLike {
+    function stop() external;
+}
+
+interface SplitterLike {
+    function wards(address) external view returns (uint256);
+    function hop() external view returns (uint256);
+}
+
+/// @title Splitter Stop Emergency Spell
+/// @notice Will disable the Splitter (Smart Burn Engine, former Flap auctions)
+/// @custom:authors [Oddaf]
+/// @custom:reviewers []
+/// @custom:auditors []
+/// @custom:bounties []
+contract SplitterStopSpell is DssEmergencySpell {
+    string public constant override description = "Emergency Spell | Disable Splitter";
```

Comment:
Nit: should we use "stop" instead of "disable" here?
We've been using the same verbs as the method names.
```suggestion
    string public constant override description = "Emergency Spell | Stop Splitter";
```